Heleonix.Reflection/Reflector.cs

```diff
@@ -975,5 +978,35 @@ private static bool GetNextContainer(MemberInfo[] memberInfos, ref object contai
 
             return false;
         }
+
+        private static object CoerceValue(Type memberType, object value)
+        {
+            if (value is null)
```

Comment:
I think we should allow to set null values.

Reply:
true, can fix